examples/llama-bench/llama-bench.cpp

```diff
@@ -508,6 +539,7 @@ static std::vector<cmd_params_instance> get_cmd_params_instances(const cmd_param
     for (const auto & tv : params.type_v)
     for (const auto & mmq : params.mul_mat_q)
     for (const auto & nkvo : params.no_kv_offload)
+    for (const auto & mmp : params.use_mmap)
```

Comment:
One minor thing, move this up, next to the other model parameters (after `tensor_split` for example), to minimize the number of times the model needs to be reloaded.

Reply:
Yes.